kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/model/Clazz.kt

```diff
@@ -0,0 +1,22 @@
+package godot.entrygenerator.model
+
+import godot.entrygenerator.ext.hasAnnotation
+
+open class Clazz(
+    open val fqName: String,
+    open val supertypes: List<Clazz> = emptyList(),
```

Comment:
I guess it contains the list of ancestors in order from child to parent.
Is that to avoid the use of a recursive function in the case you only store the direct parent?

Reply:
Yes. As i would need such a recursive function to be executed at multiple places.
I thought about making it lazy like I've done for stuff like the default value extraction, but decided that it was not worth it as i already have that information in the `godot-kotlin-symbol-processor`.
But if you think a recursive function makes more sense, we can discuss this of course

Reply:
No, no. It was just to be sure. We already got stackoverflow errors because of recursive functions, so it's nice to avoid them.
I was just wondering if it wasn't something like storing everything the class inherits/implements (so also interfaces).

Reply:
No it does not contain interfaces as currently we don't really care about them: https://github.com/utopia-rise/godot-kotlin-jvm/blob/ed18f34f64316d8e0c022eccca551531dff3f043/kt/entry-generation/godot-kotlin-symbol-processor/src/main/kotlin/godot/annotation/processor/ext/ksClassDeclarationExt.kt#L45